cernopendata/modules/fixtures/data/articles/cms-guide-for-errors/errors.md

```diff
@@ -0,0 +1,23 @@
+**I have trouble installing Virtualbox**
```

Comment:
If we go for the filename-represents-slug technique discussed elsewhere, then let's not call this "cms-guide-for-errors" 😄 but rather something like "cms-guide-troubleshooting".  @sefeg @daslerr can advise on the slugs.

(IMO it would be hepful to construct the full table like:

content | old URL | new URL
----------|--------|--------------
VMs for ALICE | `/VM/ALICE` | `/articles/virtual-machines-alice`
CMS guide written by Achim | *new content* | `/articles/cms-guide-for-research`

which can help to design the full URL slug map...)

Reply:
cms-guide-troubleshooting sounds really great!

Reply:
👍